tilequeue/query/rawr.py

```diff
@@ -703,6 +740,30 @@ def _parse_row(self, zoom, unpadded_bounds, bbox, source, fid, shape,
             if layer_name in label_layers:
                 generate_label_placement = True
 
+        # nasty hack: in the SQL, we don't query the lines table for
+        # boundary features - instead we take the boundaries of the
```

Comment:
In this case "boundary" is the outer ring of a polygon with possible hole(s)?

Reply:
Ah... thanks for pointing that out! I've used "boundary" to refer to both the features in the `boundaries` layer _and_ the line separating the interior and exterior of the polygon. I tried to make the comment clearer in 7ee2f61, please could you take a look and let me know if that has helped?